provider/cmd/run.go

```diff
@@ -239,6 +240,11 @@ func RunCmd() *cobra.Command {
 	if err := viper.BindPFlag(FlagDeploymentRuntimeClass, cmd.Flags().Lookup(FlagDeploymentRuntimeClass)); err != nil {
 		return nil
 	}
+
+	cmd.Flags().Duration(FlagBidTimeout, 20*time.Minute, "time after which bids are cancelled if no lease is created")
```

Comment:
how about 5 minutes?

Reply:
Yeah I can change the default, I just picked something